python/minmaxmap.py

```diff
@@ -14,9 +16,24 @@
     line = line.strip()
     line = line.split()
     slice_file = os.path.join(line[0])
+    slice = None
 
     #load nifti image
-    slice = nib.load(slice_file)
+    if not os.path.exists(slice_file):
+        fh = None
+
+        client = Config().get_client()
+
+        with client.read(slice_file) as reader:
+            #temporary non-recommended solution to determining if file is compressed. 
+            if 'gz' in slice_file[-2:]: 
```

Comment:
does this check that the file name *ends with* "gz"?

btw, you should test for ".gz".